### DISCUSSION TITLE
PHP 7, Chamilo 1.11.x. Chamilo 2

### DISCUSSION THREAD

**Discussion by jose867** [chamilo/chamilo-lms#5165] [General]
Hola! tengo un par de servidores funcionando con Chamilo 1.11.26, a pleno rendimiento, pero empiezo a estar preocupado por el fin de soporte de PHP 7 y la falta de actualizaciones. Entiendo que el nuevo Chamilo 2 funcionará con PHP 8, y instalación de prueba que he hecho me ha gustado. Para cuando tienen previsto el lanzamiento? Solo agradecerles el gran trabajo que hacen con este software.

Hi! I have a couple of servers running Chamilo 1.11.26, at full performance, but I'm starting to be worried about the end of PHP 7 support and the lack of updates. I understand that the new Chamilo 2 will work with PHP 8, and the test installation I've done has pleased me. When do you plan to release it? Just to thank you for the great work you do with this software.

**Comment by ywarnier** [comment]
@jose867 gracias por los agradecimientos.

No hay fecha oficial para la publicación de Chamilo 2.0. Todavía tenemos mucho por hacer, pero esperamos lanzar la alpha antes del fin de marzo, y de ahí un par de meses para la beta y otro par de meses para la estable. Estamos corriendo con el tiempo por el tema de PHP 8. Ojo que estas son estimaciones, no son una promesa.

Por si a caso, es posible, extra-oficialmente, correr Chamilo 1.11 con PHP8.1 pero hay que lanzar composer update y se pierden algunas funcionalidades (creo que la generación de gráficos falla, si bien recuerdo).

Ojo que la 2.0 *no será* recomendada para actualizaciones, solo para nuevas instalaciones desde cero. La 2.1 (fines del 2024?) debería tener las funcionalidades faltantes para hacer actualizaciones recubriendo la mayoría del espacio funcional de 1.11.26.

**Reply by jose867** [reply]
Gracias, Yannick, por la respuesta. En principio no debería suponer un problema, ya que no solemos relanzar cursos, sino ofrecer nuevos y/o actualizados (casi que redactados desde cero). Los y las usuari@s tenemos que borrarlos regularmente para cumplir con la LGPD, por lo que una instalación limpia me vale. Estaré pendiente. Saludos y gracias otra vez
